src/content/h1/info/readme.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hard-coded Data
```

Comment:
It's preferable to have the page slug (the end of the URL and directory name) match the title more closely. This aids with unique cross-site linking since all page slugs are implied link references. For example, this:
```
Click [here][hardcoded-data] to learn more.
```

is better than this:
```
Click [here][info] to learn more.
```